projects/ultrajson/README.md

```diff
@@ -30,42 +40,58 @@ Change directory to `/fuzz`:
 
 ### hypothesis_structured_fuzzer
 
-Run fuzzing:
+#### Run fuzzing:
```

Comment:
Fuzzing process could be described with only one example, just enumerate steps: Run fuzzing, minimize corpus, Get coverage, Analyze crashes.

Reply:
Also, could you rename all tomls in format: <name>-<fuzzer>.toml (for ex. json_differential_fuzzer-atheris.toml, ujson_fuzzer-pyafl.toml)